.pylintrc

```diff
@@ -0,0 +1,407 @@
+[MASTER]
```

Comment:
Do we add this into the Python27 directory? If so, please share this to Google Drive rather than committing this to GitHub